src/tests/tribol_mfem_mortar_lm.cpp

```diff
@@ -255,3 +264,84 @@ int main(int argc, char* argv[])
 
   return result;
 }
+
+void clearInactivePressureDofs(
```

Comment:
Sounds like you are going to move this to a central location for use by Tribol when using mortar methods with MFEM API. I don't think we need to expose this to the user, but it does sound like a needed utility function.